src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const mainImage = document.querySelector('img');
+const list = document.querySelector('ul');
+
+list.addEventListener('click', (event) => {
```

Comment:
It will fix eslint error
```suggestion
list.addEventListener('click', (e) => {
```

Reply:
thanks)